### PR TITLE
Make `cargo clippy --workspace --all-features` come clean

### PR DESCRIPTION
**Pull request by imp**
While adding `uuid` support I've noticed that `clippy` is not happy and took the liberty of fixing all the `clippy` suggestions.
I believe that these changes are trivial enough to be non-controversial. Feel free to reject this PR if you feel otherwise.
I tried to organize all of the fixes in "one-clippy-change-per-commit" manner to allow for cherry-picking if necessary.

### CODE REVIEW

File: macros/src/openapi/mod.rs [hunk 1]
```diff
@@ -202,16 +202,14 @@ fn quote_mediatype(m: &MediaType) -> Expr {
...
-        ObjectOrReference::Ref { ref_path: r } => r
-            .parse::<TokenStream>()
-            .expect("failed to lex path to type"),
-        ObjectOrReference::Object(schema) => match &schema.ref_path {
-            r => r
-                .parse::<TokenStream>()
-                .expect("failed to lex path to type"),
-        },
-    }
+    let r#ref = match ros {
```
Comment (kdy1) [inline macros/src/openapi/mod.rs:205]: I'm not sure if name of a variable worth raw identifier, but it's not a big issue I think.

**Review by kdy1 — APPROVED** [review]
Great! Thanks!

File: macros/src/openapi/mod.rs (hunk 1 above)
Comment (imp) [reply macros/src/openapi/mod.rs:205]: Yeah, I don't mind dropping raw identifier in favor of some other name. `ref_path`?

Comment (imp) [reply macros/src/openapi/mod.rs:205]: Oh, I see that you've already merged it. Anyway, dutifully noted.

**Pull request merged**